py/deploy.py

```diff
@@ -45,7 +45,7 @@ def setup(args):
                 ],
               },
           # TODO(jlewi): Stop pinning GKE version once 1.8 becomes the default.
-          "initialClusterVersion": "1.8.1-gke.1",
+          "initialClusterVersion": "1.7.11-gke.1",
```

Comment:
We should be using 1.8. for CRD and GPU support.

Reply:
Cool it looks from `gcloud container get-server-config` that '1.8.5-gke.0' and '1.8.4-gke.1' are the two 1.8+ versions supported. Preference?